src/Loggable/Mapping/Event/LoggableAdapter.php

```diff
@@ -2,36 +2,36 @@
 
 namespace Gedmo\Loggable\Mapping\Event;
 
+use Doctrine\Persistence\Mapping\ClassMetadata;
 use Gedmo\Mapping\Event\AdapterInterface;
 
 /**
- * Doctrine event adapter interface
- * for Loggable behavior
+ * Doctrine event adapter for the Loggable extension.
  *
  * @author Gediminas Morkevicius <gediminas.morkevicius@gmail.com>
  * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
  */
 interface LoggableAdapter extends AdapterInterface
 {
     /**
-     * Get default LogEntry class used to store the logs
+     * Get the default object class name used to store the log entries.
      *
-     * @return string
+     * @return class-string
```

Comment:
IMO, in order to keep the "native" docblocks clean, we should use the `phpstan-` prefix for cases like "class-string", "array-key", generics, etc.
```suggestion
     * @return string
     *
     * @phpstan-return class-string
```

Reply:
Covered in the rebase